de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/payment/impl/PaySelectionUpdater.java

```diff
@@ -673,6 +707,17 @@ private void setTrxName(final String trxName)
 		_trxName = trxName;
 	}
 
+	/**
+	 * Invalidates Cache for current PaySelection.
+	 */
+	private void cacheInvalidationForCurrentPaySelection()
+	{
+		final IModelCacheInvalidationService modelCacheInvalidationService = Services.get(IModelCacheInvalidationService.class);
+		modelCacheInvalidationService.invalidate(
+				CacheInvalidateMultiRequest.fromTableNameAndRecordId(I_C_PaySelection.Table_Name, _paySelection.getC_PaySelection_ID()),
```

Comment:
pls `getC_PaySelection_ID()` method instead of `_paySelection.getC_PaySelection_ID()`.

NOTE: in metasfresh, usually class fields which are prefixed with `_` shall not be used directly.